src/fullscreen/fullscreen-controller.js

```diff
@@ -164,7 +164,14 @@ class FullscreenController {
       this._player.exitPictureInPicture();
     }
     Promise.resolve(this._nativeEnterFullScreen(fullScreenElement)).then(
-      () => (this._isInFullscreen = true),
+      () => {
+        this._isInFullscreen = true;
+        const screenOrientation = screen && screen.orientation && typeof screen.orientation.lock === 'function';
```

Comment:
we can do this once in the top of the module - this check value doesn't change.

Reply:
There isn't a good way to check it... `any` is the default that could be any value, just keeping the state like it was.
The way to check it to listen to orientation change and keep an indicator if it changed not on fullscreen call.